sigstore-java/src/main/java/dev/sigstore/bundle/BundleVerifier.java

```diff
@@ -15,24 +15,125 @@
  */
 package dev.sigstore.bundle;
 
+import com.google.api.FieldBehavior;
+import com.google.api.FieldBehaviorProto;
+import com.google.protobuf.Descriptors;
+import com.google.protobuf.InvalidProtocolBufferException;
+import com.google.protobuf.MessageOrBuilder;
+import com.google.protobuf.util.JsonFormat;
 import dev.sigstore.proto.bundle.v1.Bundle;
+import java.util.ArrayList;
 import java.util.List;
+import java.util.Map;
 
-/**
- * Verifies if Sigstore Bundle is valid.
- *
- * @see <a href="https://github.com/sigstore/protobuf-specs">Sigstore Bundle Protobuf
- *     specifications</a>
- */
+/** Implements Sigstore Bundle verification. */
 public class BundleVerifier {
+  static final JsonFormat.Parser JSON_PARSER = JsonFormat.parser();
+
   /**
-   * Verify if all required fields are initialized in the input protobuf message.
+   * Parses Sigstore Bundle JSON into protobuf.
    *
-   * @param bundleJson message to verify
-   * @return list of all the missing fields
+   * @param bundleJson input JSON
+   * @return parsed bundle
+   * @throws IllegalArgumentException if the JSON is invalid
    */
-  public static List<String> findMissingFields(String bundleJson) {
-    Bundle.Builder bundle = BundleVerifierInternal.parseSigstoreBundle(bundleJson);
-    return BundleVerifierInternal.findMissingFields(bundle);
+  public static List<String> allMissingFields(String bundleJson) {
```

Comment:
a parsed bundle is returned as List<String> ?

Reply:
So this is an old method that was just moved around. The returned list is required fields that were missing. Apparently protobuf doesn't provide this functionality out of the box.

Reply:
javadoc @return needs updating I think.

Reply:
done